homeassistant/components/yolink/services.yaml

```diff
@@ -0,0 +1,43 @@
+# SpeakerHub service
+play_on_speaker_hub:
+  fields:
+    target_device:
+      required: true
+      selector:
+        device:
+          filter:
+            - integration: yolink
+              manufacturer: YoLink
+              model: SpeakerHub
+
+    message:
+      required: true
+      example: hello, yolink
+      selector:
+        text:
+    tone:
+      required: true
+      default: "Tip"
+      selector:
+        select:
+          options:
+            - "Emergency"
+            - "Alert"
+            - "Warn"
+            - "Tip"
```

Comment:
So what I was wondering, I think we can make these snake_case and provide translations for them

Reply:
Where can I find examples of options translations?

Reply:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/media_player/strings.json#L319-L333 in combination with https://github.com/home-assistant/core/blob/dev/homeassistant/components/media_player/strings.json#L319-L333

Reply:
> https://github.com/home-assistant/core/blob/dev/homeassistant/components/media_player/strings.json#L319-L333 in combination with https://github.com/home-assistant/core/blob/dev/homeassistant/components/media_player/strings.json#L319-L333

thanks, i got it